lib/std/fmt.zig

```diff
@@ -1321,6 +1322,11 @@ test "enum" {
     try testFmt("enum: Enum.Two\n", "enum: {}\n", .{&value});
     try testFmt("enum: Enum.One\n", "enum: {x}\n", .{Enum.One});
     try testFmt("enum: Enum.Two\n", "enum: {X}\n", .{Enum.Two});
+
+    // test very large enum to verify ct branch quota is large enough
+    if (builtin.os.tag == .windows) {
```

Comment:
the enum should exist even when not targeting windows

Reply:
fixed